### ISSUE TITLE
Clean up directory structure & convert notebooks to .md files

### ISSUE THREAD

**Issue by dhavide** [ScienceCore/climaterisk#59]
Roughly speaking, this is what is needed:

+ Convert all existing notebook (i.e., `.ipynb`) files to Markdown(`.md`) files, e.g.,
  ```bash
  jupytext notebook.ipynb --output notebook.md
  # Equivalently,
  # jupytext notebook.ipynb -o notebook.md
  ```
  The current `environment.yaml` file includes `jupytext` so try updating your environment if `jupytext` is not installed locally.

  To be safe, make sure that the files are converted correctly by doing a round-trip conversion and using `diff` to ensure that the original `.ipynb` file is restored correctly. There will be some differences (e.g., in cell labels) but those can safely be ignored.

+ Introduce a directory structure that looks something like this:
  ![Untitled](https://github.com/ScienceCore/climaterisk/assets/2997697/a427cb61-d039-4852-9fa3-c7d5411f4beb)
  (use numbered directories in sequence according to the diagram so that the intended order is clear, e.g., `01_Open_Science/`, `02_Geospatial_fundamentals/`, etc. Use the sequence as in the diagram for now; that may change later).
  Move MarkDown files (and whatever else) into the appropriate directories. This may break relative links to, e.g., image files in the `assets` folder. Check to make sure these links are correct before checking in.

+ Use a directory called, e.g., `scratch` to store files that currently do not have an obvious place; they most likely have a future use.

**Comment by jnywong** [comment]
Heya 👋 Let me know if I can help with this one @JFormoso  – seems like a pretty fundamental infrastructure issue that needs attention before work on #40 can start.

**Comment by JFormoso** [comment]
Hi @dhavide! This is what I got so far, I am not sure about the location of many of the files, such as the nb to select AOI, and remote sensing, which I included in geospatial fundamentals. Also, there are many notebooks that seem to be placeholders.
This is the current structure:

│   .gitignore
│   apt.txt
│   CONDUCT.md
│   CONTRIBUTING.md
│   environment.yml
│   LICENSE
│   README.md
│   requirements.txt
│
├───.github
│   └───workflows
│           build.yml
│           deploy.yml
│
├───assets
│       2i2c_login.png
│       2i2c_logo.png
│       banner.jpg
│       banner_with_person.jpg
│       climate_risks_cloud_data_filled.png
│       climate_risks_cloud_data_filled.svg
│       climate_risks_cloud_data_line.png
│       climate_risks_cloud_data_line.svg
│       climate_risks_drought_color.png
│       climate_risks_drought_filled.png
│       climate_risks_drought_filled.svg
│       climate_risks_drought_line.png
│       climate_risks_drought_line.svg
│       climate_risks_fire_color.png
│       climate_risks_fire_filled.png
│       climate_risks_fire_filled.svg
│       climate_risks_fire_line.png
│       climate_risks_fire_line.svg
│       climate_risks_flood_color.png
│       climate_risks_flood_filled.png
│       climate_risks_flood_filled.svg
│       climate_risks_flood_line.png
│       climate_risks_flood_line.svg
│       climate_risks_remote_sensing_filled.png
│       climate_risks_remote_sensing_filled.svg
│       climate_risks_remote_sensing_line.png
│       climate_risks_remote_sensing_line.svg
│       earthdata_login.png
│       earthdata_profile2.png
│       earthdata_profile_old.png
│       image165.png
│       image377.jpg
│       image5.png
│       image7.jpeg
│       logo.png
│       MD_logo.png
│       points-lines-polygons-vector-data-types.png
│       SETUP1.png
│       start_server.png
│       TOPS.png
│       work_environment_jupyter_lab.png
│
└───book
    │   _config.yml
    │   _toc.yml
    │
    ├───01_Open_Science
    │       Open_Science_Intro.md
    │       Open_Science_Intro_Slides.md
    │
    ├───02_Geospatial_fundamentals
    │       2_Selecting_an_AOI.md
    │       remote-sensing.md
    │
    ├───03_Geospatial_data_files
    │       geographic_data_formats.md
    │
    ├───04_NASA_Earthdata
    │       0_Configuracion_inicial.md
    │       0_Initial_Setup.md
    │       1_Getting_Started.md
    │       earthdata-cloud.md
    │
    ├───05_2i2c_Hub
    ├───06_Python_APIs
    ├───07_Wildfire_analysis
    │       Retrieving_Disturbance_Data.md
    │       Wildfire.md
    │
    ├───08_Flood_analysis
    │       3_Retrieving_FloodData.md
    │       flood.md
    │
    └───09_Scratch
        │   .netrc
        │   4_Analyzing_Datasets.md
        │   5_Manipulating_and_Visualizing_Datasets.md
        │   drought.md
        │   intro.md
        │   proposal.md
        │   references.bib
        │   SLIDES-NASA-TOPS-flood-EN.md
        │   SLIDES-NASA-TOPS-flood-ES.md
        │
        ├───about_the_tutorial
        │       Assessment_Activities.md
        │       disenio_leccion.md
        │       Learner_Personas.md
        │       Module_Objectives.md
        │       Outcomes.md
        │       Requirements.md
        │
        └───notebooks
                2_ES_Flood.md

Let me know what I need to change. 

Regarding the files, I converted all files to .md, did the round-trip conversion, use diff to check if the conversion was done properly.

**Comment by PatriLoto** [comment]
Hi @JFormoso and @dhavide.I just wanted to let you know that the files **1_Getting_Started_text.md**
and **earthdata-cloud.md** are no longer needed because their contents are included in the **Initial_setup.md** file.

**Comment by JFormoso** [comment]
Thanks Patri, I'll correct that!

**Comment by jnywong** [comment]
Hey @JFormoso I'm happy with this directory structure! The _assets_ directory will need to move into the _book_ folder for the Jupyter Book to find the image files however.

Can I double-check that the `01_Open_Science/Open_Science_Intro.md` will now serve as the "homepage" for the tutorial?

**Comment by JFormoso** [comment]
Hi @jnywong I really don't know if the intro.md file is the future homepage. All ask at Friday's meeting. I made the changes you suggested, I am not sure if the pull request shows them or how to merge that. Let me know!

**Comment by jnywong** [comment]
Closed by pull requests #70 and #73
